catalyst/dl/runner/gan_runner.py

```diff
@@ -0,0 +1,71 @@
+from typing import Mapping, Any
+
+import torch
+from catalyst.dl import Runner
+
+
+class GANRunner(Runner):
+    def __init__(
+        self,
+        model=None,
+        device=None,
+        images_key="images",
```

Comment:
to make it more general, let's use `features` as a key?

Reply:
ok